pages/login.tsx

```diff
@@ -20,12 +20,30 @@ export default function Login() {
     signIn('email', { email });
   };
 
+  /**
+   * Log in with email when the 'Enter' key is pressed
+   */
+  const handleKeyUp = (event: KeyboardEvent) => {
+    if (event.key === 'Enter') {
+      signInWithEmail();
+    }
+  };
+
   return (
     <Layout>
       <Container>
-        <Heading>Login</Heading>
-        <Input value={email} onChange={event => setEmail(event.target.value)} />
-        <Button onClick={signInWithEmail}>Log in</Button>{' '}
+        <FormControl>
+          <FormLabel>Login</FormLabel>
+          <Input
+            type="email"
+            placeholder="Your email"
+            value={email}
+            onChange={event => setEmail(event.target.value)}
+            onKeyUp={handleKeyUp}
+          />
+          <FormHelperText>Please enter an RCD email</FormHelperText>
+        </FormControl>
+        <Button onClick={signInWithEmail}>Log in</Button>
```

Comment:
Can we refactor this slightly? I think we can wrap the entire thing up in a form like this, without having to write an onKeyUp fn to support enter key submit.

```
<form>
  <FormControl>
        ....
  </FormControl>
  <Button onClick={() => signInWithEmail}>Log in</Button>
</form>
```